### DISCUSSION TITLE
Green bar above DSB box

### DISCUSSION THREAD

**Discussion by danielebruneo** [df8oe/UHSDR#1948] [General]
Can someone explain what that green bar above the DSP box is for?
Couldn't find any info on that.

**Comment by df8oe** [comment]
I do not see a green bar there. Not on 2.8" and not on 3.5" LCD.

**Comment by danielebruneo** [comment]
on the mid left side of the screen there is a like a green rectangle with a cursor swinging left and right...

**Comment by df8oe** [comment]
OK - we thought this is self-explaining... It is a center meter which is not shown in all modes. Especially in CW decoding works best if frequency is tuned best.

**Reply by danielebruneo** [reply]
Makes sense... thanks for making that clear to me.
